### DISCUSSION TITLE
Status JAXi implementation

### DISCUSSION THREAD

**Discussion by JoostBaars** [OE4T/meta-tegra#1256] [General]
I saw in [this ](https://github.com/OE4T/meta-tegra/discussions/1217) thread that JAXi support was already implemented or being worked on, but I didn't see the support in the README yet. So what's the current status of JAXi support for the kirkstone branch?

I've added JAXi support for the kirkstone branch locally. So if it's not added yet, I can also create a pull-request that adds JAXi support.

**Comment by ichergui** [comment]
Hey @JoostBaars 
Welcome to the OE4T community.

Please submit PR and we will review it. Thanks
